ui/shared/Revision.jsx

```diff
@@ -118,7 +118,8 @@ export class Revision extends React.PureComponent {
                 </div>
               );
             })}
-          {comment}
+          {<div style={{ visibility: 'hidden' }}>invisible space</div>}
```

Comment:
Rather than this, perhaps put the `comment` in a span with a `className="mt-3"`  This will set `margin-top` to `3rem`.

Reply:
Hey Cam, I applied what I think you described, and it doesn't seem that the styling works inside the tooltip

Reply:
Ahh, you're right!  OK, how about adding `mb-3` (or 4) to the `<div>` that contains the bug summary?  And I was thinking it might look nicer to have the word `Commit:` on a separate line above the commit message like this:

    Commit:
    Bug XXXX - This commit message

So, you can just put "Commit:" as a string, and put the `{comment}` in a `<div>`.  Would you give that a try?

It will look like this:

![Screenshot 2020-07-30 12 33 07](https://user-images.githubusercontent.com/419924/88966282-e6420080-d260-11ea-873b-ea2cd47e5888.png)